bin/beaker

```diff
@@ -3,7 +3,11 @@
 require 'rubygems' unless defined?(Gem)
 require 'beaker'
 
-Beaker::CLI.new.execute!
-puts "Beaker completed successfully, thanks."
+if execute_subcommand?(ARGV[0])
```

Comment:
@ferglor I've figured it out and written it up in [BKR-1026](https://tickets.puppetlabs.com/browse/BKR-1026). I think that this means we should think about reimplementing the location of shared subcommands so they don't get loaded into the top level namespace.